integrationTests/vm/testInitializer.go

```diff
@@ -42,6 +42,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process/smartContract"
 	"github.com/ElrondNetwork/elrond-go/process/smartContract/builtInFunctions"
 	"github.com/ElrondNetwork/elrond-go/process/smartContract/hooks"
+	syncDisabled "github.com/ElrondNetwork/elrond-go/process/sync/disabled"
```

Comment:
disabledSync sounds bettter

Reply:
we use this pattern in more places so I wouldn't change it.